src/app/modules/users/single-user/single-user.component.ts

```diff
@@ -114,13 +241,16 @@ export class SingleUserComponent implements OnInit {
   * To Create the User
   */
   createUser(userdata) {
+    console.log('createUser', userdata);
     let data = {
```

Comment:
why are you creating an object like this? form values should give you the correct object

Reply:
Yes, i removed